draft-ietf-privacypass-protocol.md

```diff
@@ -380,6 +415,30 @@ opaque PublicKey<1..2^16-1>
 opaque PrivateKey<1..2^16-1>
 ~~~
 
+### CommitRequest {#pp-cli-commit-request}
+
+The `CommitRequest` struct is simply a fixed message corresponding to
+the ID of the server.
+
+~~~
+struct {
+  opaque server_id<1..2^16-1>
```

Comment:
Interesting -- do we need this? Can it be empty?

Reply:
I was allowing for the possibility that the client may only want to allow commitments for certain types (e.g. it may only want to allow server S with ciphersuite `c1`, rather than `c2`). The `server_id` was kind of a placeholder for that. However, it is pretty ugly right now, so I would be happy to remove it.

Reply:
👍 Eh, it's probably fine for now.